daffodil-cli/src/main/scala/org/apache/daffodil/Main.scala

```diff
@@ -502,11 +502,47 @@ class CLIConf(arguments: Array[String]) extends scallop.ScallopConf(arguments)
     val info = tally(descr = "increment test result information output level, one level for each -i")
   }
 
+  // Generate Subcommand Options
+  object generate extends scallop.Subcommand("generate") {
+    descr("generate <language> code from a DFDL schema")
+
+    banner("""|Usage: daffodil [GLOBAL_OPTS] generate <language> [SUBCOMMAND_OPTS]
+              |""".stripMargin)
+    shortSubcommandsHelp()
+    footer("""|
+              |Run 'daffodil generate <language> --help' for subcommand specific options""".stripMargin)
+
+    object c extends scallop.Subcommand("c") {
```

Comment:
"c-runtime2" would be good. There's definitely no reason to be trying to describe characteristics of the C code (thread safe or not etc.) in these identifiers, but the kind of thing being generated is C code, with runtime 2 in mind. 
I'm ignoring that "runtime2" is itself a bit challenged as a name, but I've long believed that sometimes the best distinction between things is just "add a number" because trying to capture description in a short identifier is hard.

Reply:
"c-runtime2" makes me think there's a c-runtime1. Maybe just plain "c" is reasonable, considering [DAFFODIL-2406](https://issues.apache.org/jira/browse/DAFFODIL-2406) suggests renaming things to "backend-c-generator". The "generator" part is assumed in the "daffodil generator" subcommand, and backend is also implied. So maybe just plain "c" is the best option?

Reply:
Agree that "c" is fine for now. No change required here.